.github/workflows/legacy-release_sbom-generator.yaml

```
@@ -11,7 +11,7 @@ on:
 
 jobs:
   scan:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Why not use the variable here with fallback?

```suggestion
    runs-on: ubuntu-${{ vars.UBUNTU_RUNNER_VERSION || '22.04' }}
```

---

I think this workflow should be removed. It's deprecated. The SBOM generation is now an Action, and it's called from the release process.